utils/kernel.c

```diff
@@ -491,7 +494,8 @@ static int reset_tracing_files(void)
 	return 0;
 }
 
-static int __setup_kernel_tracing(struct uftrace_kernel_writer *kernel)
+static int __setup_kernel_tracing(struct uftrace_kernel_writer *kernel,
+                                  struct opts *opts)
```

Comment:
Can you please add a `clock` field in the `struct uftrace_kernel_writer` and use it instead?  I'd like to have all info to setup the kernel tracing in the struct.